src/CMakeLists.txt

```diff
@@ -595,11 +597,10 @@ message("\n")
 # INSTALL RULES
 # ==============================================================================
 
-install(EXPORT openMVG-targets DESTINATION lib)
 install(EXPORT openMVG-targets
   FILE OpenMVGTargets.cmake
   NAMESPACE OpenMVG::
-  DESTINATION share/openMVG/cmake
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/openMVG/cmake
```

Comment:
I would still keep the share here (see cmake doc https://cmake.org/cmake/help/book/mastering-cmake/chapter/Install.html)

Reply:
So lib/share would be an appropriate path from your knowledge. We can check the behavior on a local folder install and see if consistent with the doc https://github.com/openMVG/openMVG/blob/develop/BUILD.md#using-openmvg-as-a-third-party-library-dependency-with-cmake

Reply:
If you meant `lib` _or_ `share`, then yes, but not `lib/share` (unless `lib/` is actually part of the INSTALL_PREFIX).

(`|` in the link above is part of a regex and means "or").

But `share` is only allowed for architecture independent packages (e.g. header only libraries).

Reply:
Convinced this is correct and does not cause any breakage?

Reply:
In the documentation, `<userprefix>/share/cmake` should become `<userprefix>/lib/cmake`.

(In case `CMAKE_INSTALL_PREFIX` is not one of the standard system prefixes, e.g. `/usr/` or `/usr/local/`, CMake defaults to just `lib`.)